app/api/helpers/mail.py

```diff
@@ -329,6 +329,23 @@ def send_email_to_attendees(order):
     if current_app.config['ATTACH_ORDER_PDF']:
         attachments = [order.ticket_pdf_path, order.invoice_pdf_path]
 
+    event = order.event
+    event_id = event.id
+    filedir = os.path.join(
+        current_app.config.get('BASE_DIR'),
+        f'static/uploads/{event_id}/',
+    )
+
+    if not os.path.isdir(filedir):
+        os.makedirs(filedir)
+    filename = "ical.ics"
+    ical_file_path = os.path.join(filedir, filename)
+    if ical_file_path:
+        if attachments == None:
+            attachments = [ical_file_path]
+        else:
+            attachments.append(ical_file_path)
```

Comment:
Just generate on the go

Reply:
We can surely do that, won't it be a bad practice since, there'd be multiple copies of the same file. As it already generates a `.ics` file when an event is created.